wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceRuntimes.java

```diff
@@ -643,7 +650,8 @@ public void run() {
   }
 
   private void subscribeAbnormalRuntimeStopListener() {
-    eventService.subscribe(new AbnormalRuntimeStopListener());
+    eventService.subscribe(new AbnormalRuntimeStoppingListener());
```

Comment:
Shouldn't we unsubscribe them somewhere?

Reply:
We used to `unsubscribe` listener on `@PreDestroy` phase if it is subscribed on `@PostConstruct` but then decided that it is not necessary to do since Che Server is going to be stopped.

Reply:
I forgot that is it a global component, not per workspace. Thanks for the explanation